### DISCUSSION TITLE
Terrakube atlantis integration

### DISCUSSION THREAD

**Discussion by gdacal-gocloud** [terrakube-io/terrakube#1030] [Q&A]
Is it possible to integrate atlantis with terrakube?

**Comment by alfespa17** [comment]
I never tested that but I guess it could work with something similar to [this](https://www.runatlantis.io/docs/terraform-cloud.html)

**Comment by stanleyz** [comment]
Atlantis requires a server application to be maintained, as a customer I will not want to manage two IaC tools and integrate them, would it be possible to build the functionalities within Terrakube? Thus customers would just pick one tool.
